contracts/interfaces/ICCIPCollector.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.5.16;
```

Comment:
is it must this low sol version not 0.8.0?

Reply:
it's due to StakingThales being 0.5.16